torch_xla/csrc/runtime/computation_client.cc

```diff
@@ -53,6 +53,21 @@ int64_t ComputationClient::GetDeviceOrdinal(const std::string& device) {
   return std::stoi(device.substr(pos + 1));
 }
 
+std::unordered_map<std::string, std::string> pjrt_plugins_;
```

Comment:
I thought user can only register one plug in? What;s the use case of registering multiple?

Reply:
We'll register TPU and GPU as default options, and then other packages will add plugins on top of those. JAX is also using [Python entry points](https://packaging.python.org/en/latest/specifications/entry-points/) to register available plugins automatically, which we may also want to do.